src/openeo_aggregator/utils.py

```diff
@@ -110,6 +110,37 @@ def union(self, key: str, skip_duplicates=False) -> list:
                 result.append(item)
         return result
 
+    def simple_merge(self, included_keys=None) -> dict:
+        """
```

Comment:
this is called "simple" but the explanation and implementation is not that obvious actually

Reply:
I managed to eliminate simple_merge and move the logic closer to the actual collection metadata fields